smt/applications/ego.py

```diff
@@ -99,6 +99,8 @@ def _initialize(self):
         declare("ydoe", None, types=np.ndarray, desc="Initial doe outputs")
         declare("xlimits", None, types=np.ndarray, desc="Bounds of function fun inputs")
         declare("verbose", False, types=bool, desc="Print computation information")
+        declare("tunnel", 0, types=int, desc="1 to enable tunneling in ei")
+        declare("surrogate", KRG(print_global=False), desc="surrogate model to use")
```

Comment:
I think you can declare types=KrgBased or more strictly types=(KRG, KPLS, KPLSK) maybe description could be more specific "SMT kriging-based surrogate model used internally"